metecho/api/tests/serializers.py

```diff
@@ -280,3 +280,16 @@ def test_invalid(self, rf, user_factory, task_factory, scratch_org_factory):
             data={"task": str(task.id), "org_type": "Dev"}, context={"request": r}
         )
         assert not serializer.is_valid()
+
+    def test_circumspect_ignored_changes(
+        self, rf, user_factory, task_factory, scratch_org_factory
+    ):
+        user = user_factory()
+        task = task_factory()
+        instance = scratch_org_factory(
+            task=task, org_type="Dev", owner=user, ignored_changes={"test": "value"}
+        )
+
+        r = rf.get("/")
+        serializer = ScratchOrgSerializer(instance, context={"request": r})
+        assert serializer.data["ignored_changes"] == {}
```

Comment:
@wlonk want to check my assumption that `ignored_changes` field would be updated with the user selection.  Looks like the call to update an org is successful, returning whether there are ignored changes, and it's count; however `ignored_changes` doesn't update (I think this test is asserting such). Is that expected?

Reply:
So it should update if you can see the field; `ignored_changes` is privacy-protected like `unsaved_changes`.

Reply:
hm, OK this is what the response looks like: am I missing something? 
![Screenshot from 2020-04-22 15-27-09](https://user-images.githubusercontent.com/12376362/80024941-d1114500-84ad-11ea-8853-a27220cc2853.png)